feature-detects/css/hairline.js

```diff
@@ -0,0 +1,20 @@
+/*! {
+  "name": "hairline",
+  "property": "hairline",
+  "tags": ["css"],
+  "authors": ["strarsis"],
+  "notes": [{
+    "name": "Blog post about CSS retina hairlines",
+    "href": "http://dieulot.net/css-retina-hairline"
+  },{
+    "name": "Derived from",
+    "href": "https://gist.github.com/dieulot/520a49463f6058fbc8d1"
+  }]
+}
```

Comment:
can you add a description as to what hairline is? I don't think the naming is super well known